packages/corejs/src/monitoring/index.tsx

```diff
@@ -0,0 +1,130 @@
+import * as Sentry from '@sentry/browser';
+import { BrowserTracing } from '@sentry/tracing';
+import Constants from 'expo-constants';
+import React, { createContext, useCallback, useContext, useEffect, useMemo } from 'react';
+import { Primitive } from '@sentry/types';
+
+import { MonitoringConfigType, MonitoringContextType, SentryTransactionStatus } from './types';
+
+export * from './types';
+
+/**
+ * Monitoring context which will create wrapper for monitoring functionality.
+*/
+export const MonitoringContext = createContext<MonitoringContextType | null>(null);
+
+/**
+ * Monitoring wrapper used to abstract Sentry functionality.
+ *
+ * @param {object} children - Provider children components that will be shared inside context provider.
+ * @param {MonitoringConfig} config - Configuration for sentry to override default configuration.
```

Comment:
The params provided here are actually members of an object in the function signature. So in the JSDoc they should not be displayed as function params but as object properties